app/uk/gov/hmrc/mobiletaxcreditssummary/controllers/action/Authorisation.scala

```diff
@@ -70,7 +70,7 @@ trait Authorisation extends Results with AuthorisedFunctions {
         block(request)
       }
       .recover {
-        case _: uk.gov.hmrc.http.Upstream4xxResponse =>
+        case _: uk.gov.hmrc.http.UpstreamErrorResponse =>
```

Comment:
We should add a check on this case for error code >399 <500 as it is only there to catch 400s